test/tests/functional/pbs_sched_attr_updates.py

```diff
@@ -132,9 +130,11 @@ def test_accrue_type(self):
         self.scheduler.run_scheduling_cycle()
         self.server.expect(JOB, "comment", op=SET, id=jid4)
         self.server.expect(JOB, "accrue_type", op=SET, id=jid4)
-        self.server.expect(JOB, {"eliglible_time": "00:00:00"}, op=NE, id=jid4)
-        self.server.expect(JOB, "comment", op=SET, id=jid3)
-        self.server.expect(JOB, "accrue_type", op=SET, id=jid3)
-        self.server.expect(JOB, {"eliglible_time": "00:00:00"}, op=NE, id=jid3)
-        self.server.expect(JOB, "accrue_type", op=SET, id=jid2)
-        self.server.expect(JOB, {"eliglible_time": "00:00:00"}, op=NE, id=jid2)
+        ret = self.server.status(JOB, id=jid4)
+        elig_time = ret[0]["eligible_time"]
```

Comment:
Why the change from expect() to status()?

Reply:
thanks for pointing it out. The expect was failing and I couldn't figure out why because eligible_time was correctly getting accrued. Now, finally, I see that I had a typo "eliglible" instead of "eligible", doh! I'll change it back.